docs/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Next
 
 * Collect Postmaster now requires an additional click to confirm.
+* You can drag items from existing loadouts in the Loadouts page into the loadout editor - use it to grab a couple of your favorite pieces from another loadout!
```

Comment:
```suggestion
* On the Loadouts page, you can now drag existing items on the page, into the current Loadout Editor. Use it to grab a couple of your favorite pieces from another loadout!
```
maybe? it was kind of a run-on string of prepositions. just me but maybe a good time to remind people this already works on inventory page as well.

Reply:
Yeah that's a good idea